.github/scripts/fetch-pokemon-data.py

```
@@ -7,17 +7,27 @@
+FORM_WORKERS = 8
+
 def fetch_json(url: str, max_retries: int = 3) -> dict:
     """URLからJSONデータを取得（リトライ機能付き）"""
     last_error = None
 
+    request = urllib.request.Request(url, headers={'User-Agent': USER_AGENT})
+
     for attempt in range(max_retries):
         try:
-            with urllib.request.urlopen(url) as response:
+            with urllib.request.urlopen(request) as response:
```

Ruff (0.16.0)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)